test/regress/cond4.sh

```diff
@@ -0,0 +1,8 @@
+set -xe
```

Comment:
I would just leave it as it was before without the shell script and test a single define. There's no reason to test two unless there's likely to be a bug there (I don't see why there would be).

Reply:
Well, I always try to cover every case, thats why I changed it. I can revert it, if you want, but I would preffer testing both, despite chance of bug is small here.

Reply:
Let's just test one define without the shell script. Then the test can run on Windows and there is less overhead.